java-tools/OsmAndMapCreatorUtilities/src/main/java/net/osmand/util/FilterMap.java

```diff
@@ -0,0 +1,102 @@
+package net.osmand.util;
+
+import net.osmand.binary.BinaryMapIndexReader;
+import net.osmand.map.OsmandRegions;
+import net.osmand.map.WorldRegion;
+
+import java.io.File;
+import java.util.List;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+public class FilterMap {
+    
+    FilterMap() {
+    }
+    
+    public static boolean checkBiggerMapExist(List<BinaryMapIndexReader> files, List<String> regionNameList, String fileName, OsmandRegions osmandRegions) {
```

Comment:
1. all static methods doesn't look good
2. parameter list is rather long
3. some methods should be private
4. Naming: FilterMap? Suggestion: UtilityToExcludeDuplicatedMaps

Reply:
1. `+`
2. partially
3. `+`
4. `+`